salt/modules/ceph.py

```diff
@@ -0,0 +1,654 @@
+import logging
```

Comment:
The top of the file needs a file encoding line: `# -*- coding: utf-8 -*-`

Can you also add a doc string at the top of this file describing the new module? Be sure to include a `.. versionadded:: Carbon` directive in there so people know when this was added.